src/stim/stabilizers/tableau.pybind.cc

```diff
@@ -208,6 +208,34 @@ void stim_pybind::pybind_tableau(pybind11::module &m) {
         )DOC")
             .data());
 
+    c.def(
+        "to_pauli_string",
+        [](const Tableau &self) {
+            return PyPauliString(self.to_pauli_string());
+        },
+        clean_doc_string(u8R"DOC(
+            If the tableau fixes each pauli upto sign, creates the pauli
```

Comment:
I wouldn't mention the "up to sign" thing here, since this conversion is exact. It's the pauli-to-tableau conversion that loses the sign information.

Reply:
:+1: